internal/handlers/common/params.go

```diff
@@ -65,3 +67,26 @@ func AssertType[T types.Type](value any) (T, error) {
 
 	return res, nil
 }
+
+// GetNumberParam matches value's type returning protocol error for bad float values and unexpected results.
```

Comment:
It, however, does not return protocol error.

Reply:
Yeah, it is left after some refactoring.